### DISCUSSION TITLE
Images & Emojis

### DISCUSSION THREAD

**Discussion by gh4E1NtQii** [0perationPrivacy/VoIP#154] [General]
### Issue
I was testing out VoIP Suite - I have both Render and Heroku setup with separate Telnyx numbers for testing.  Normal text works well.  Some emojis do not render and images do not display - they just say `Image`.   

### Home Firewall Setup
I do use a pfsense router with pfBlockerNG-devel and ProtonVPN.  My browser is Firefox, configured with the steps from Extreme Privacy, along with uBlock Origin.  I have also tried Brave with the same results.  I am sending the test messages from MySudo behind the firewall.  I have tried just using my cell connection and get the same result.  MySudo uses Giphy, which loads properly behind my pfsense firewall and shows the green check mark that it sent.  VoIP Suite takes a little bit of extra time but it receives it, but does not display.  Same thing occurs if I just attach an image to be sent.  Did I miss a step perhaps?  

### Screen Shot

![voipsuite_images](https://user-images.githubusercontent.com/96756919/187555610-5875b666-3af8-4eba-bd3f-dd4122499726.png)

**Comment by 0perationPrivacy** [comment]
I've moved this into the Issues section https://github.com/0perationPrivacy/VoIP/issues/155. I will troubleshoot the links and resolve it. Will require some link inspection in the network tab to see what's blocking it. Thank you.
